externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/spark/kyuubi/SparkContextHelper.scala

```diff
@@ -30,6 +32,11 @@ object SparkContextHelper {
   def createSparkHistoryLogger(sc: SparkContext): EventLogger[KyuubiSparkEvent] = {
     new SparkHistoryEventLogger(sc)
   }
+
+  def postListenerEvent(event: SparkListenerEvent): Unit = {
+    KyuubiSparkUtil.globalSparkContext.listenerBus.post(event)
+  }
+
```

Comment:
we can move this to L48, `sc.listenerBus.post(event)`

Reply:
Then EventManager seems t not need anymore

Reply:
Nice, got.